lib/web_ui/lib/src/engine/text_editing/input_type.dart

```diff
@@ -94,6 +97,17 @@ class NoTextInputType extends EngineInputType {
   String get inputmodeAttribute => 'none';
 }
 
+/// Multi-line no text input.
```

Comment:
We should probably have a clear comment description of why this is needed or in what case this type might be used